src/main/java/mate/jdbc/dao/manufacturer/ManufacturerDao.java

```diff
@@ -1,4 +1,4 @@
-package mate.jdbc.dao;
+package mate.jdbc.dao.manufacturer;
```

Comment:
it's better to put interfaces `ManufactureDao` and `DriverDao` directly in `dao` folder, and for their implementations, you could create a separate package `impl` inside of `dao` folder and put them there